### PR TITLE
feat: refactor to help users directly use OpenMLDB DB API

### PR DESCRIPTION
**Pull request by HuilinWu2**
[Python SDK] Now users are able to use OpenMLDB DB API through:

```
import openmldb as db

# get openmldb DB API
dbapi = db.dbapi

# Open database connection
connection = dbapi.connect("db_test", "127.0.0.1:6181", "/onebox")

# prepare a cursor object using cursor() method
cursor = connection.cursor()

# execute sql
cursor.execute("create database db_test;")

...

```

### CODE REVIEW

File: python/openmldb/sdk/__init__.py [hunk 1]
```diff
@@ -11,3 +11,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from . import sdk
```
Comment (tobegit3hub) [inline python/openmldb/sdk/__init__.py]: We may change `from sdk import *` later and update the usage in `dbapi.py` or others.

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: python/openmldb/dbapi/dbapi.py [hunk 2]
```diff
@@ -18,8 +18,8 @@
 sys.path.append("../")
 
 import logging
-from openmldb_sdk import sdk as sdk_module
-from openmldb_native import sql_router_sdk
+from ..sdk import sdk as sdk_module
+from ..native import sql_router_sdk
```
Comment (tobegit3hub) [inline python/openmldb/dbapi/dbapi.py]: Make sure this works in linux and macos environments.

**Comment by tobegit3hub** [comment]
We may update the docs, just like https://www.runoob.com/python3/python3-mysql.html and https://www.runoob.com/python/python-mysql.html .

```
import openmldb.dbapi

db = openmldb.dbapi.connect("db_test", "127.0.0.1:6181", "/onebox")

cursor = db.cursor()

cursor.execute("create database db_test;")

...
```

**Comment by tobegit3hub** [comment]
Add an unit test if it is possible to use dbapi directly.

File: python/openmldb/sdk/__init__.py (hunk 1 above)
Comment (HuilinWu2) [reply python/openmldb/sdk/__init__.py]: updated.

File: python/openmldb/dbapi/dbapi.py (hunk 2 above)
Comment (HuilinWu2) [reply python/openmldb/dbapi/dbapi.py]: This is relative import and should work in Linux and Macos:
* If the import module in the same dir, use e.g: from . import core
* If the import module in the top dir, use e.g: from .. import core
* If the import module in the other subdir, use e.g: from ..other import core

**Review by dl239 — APPROVED** [review]

File: python/openmldb/__init__.py [hunk 3]
```diff
@@ -13,3 +13,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from . import openmldb_dbapi
+from . import openmldb_sdk
+from . import openmldb_native
+from . import sqlalchemy_openmldb
```
Comment (vagetablechicken) [inline python/openmldb/__init__.py]: add newline

File: python/openmldb/dbapi/__init__.py [hunk 4]
```diff
@@ -11,3 +11,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from .dbapi import *
```
Comment (vagetablechicken) [inline python/openmldb/dbapi/__init__.py]: add newline

File: python/openmldb/test/sql_smoke_test.py [hunk 5]
```diff
@@ -14,7 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from openmldb_sdk import sdk as sdk_module
+import sys
```
Comment (vagetablechicken) [inline python/openmldb/test/sql_smoke_test.py:18]: Will it be reformat? You'd better add fmt:off

**Review by vagetablechicken — CHANGES_REQUESTED** [review]

File: python/openmldb/test/sql_smoke_test.py (hunk 5 above)
Comment (vagetablechicken) [reply python/openmldb/test/sql_smoke_test.py:18]: And `../` is not a good way, `sys.path.append(os.path.dirname(__file__) + "/..")` is better?

Comment (tobegit3hub) [reply python/openmldb/test/sql_smoke_test.py:18]: We don't need add "../" now for relative import. @HuilinWu2 please try to remove these.

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**